addons/humor/yeetgun/config.cpp

```diff
@@ -0,0 +1,117 @@
+class CfgPatches 
+{
+    class cnto_humor 
```

Comment:
Since this `config.cpp` is unique to yeetgun, it shouldn't hog the namespace in `CfgPatches`. Rename to `cnto_humor_yeetgun`.